.github/workflows/refresh.yml

```
@@ -1,27 +1,22 @@
 on:
   schedule:
-      - cron: "0 * * * *"
+  - cron: "0 * * * *"
   workflow_dispatch:
-  
+
 jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - name: Retrieve
-        run: |
-          while read repository; do
-            curl --silent --location "https://hub.docker.com/v2/repositories/$repository" |
-            jq .pull_count | paste <(date +%s) - >> "$(basename "$repository")"
-          done <<-END
-            dvcorg/cml-gpu-py3-cloud-runner
-            dvcorg/cml-gpu-cloud-runner
-            dvcorg/cml-py3-cloud-runner
-            dvcorg/cml-cloud-runner
-          END
-      - name: Commit
-        run: |
-          git config --local user.name "GitHub Action"
-          git config --local user.email "action@github.com"
-          git add --all && git commit --message "Update data"
-          git push
+    - uses: actions/checkout@v2
+    - name: Retrieve
+      run: |
+        for repository in gpu-py3-cloud-runner gpu-cloud-runner py3-cloud-runner cloud-runner; do
+          curl -L "https://hub.docker.com/v2/repositories/dvcorg/cml-$repository" \
```

```suggestion
          curl --location "https://hub.docker.com/v2/repositories/dvcorg/cml-$repository" \
```

**The [OCD](https://en.wikipedia.org/wiki/Obsessive%E2%80%93compulsive_disorder) I don't have** tends to prefer [GNU-style long options](https://www.gnu.org/software/libc/manual/html_node/Argument-Syntax.html) over short options. Specially on full moon nights.

---

I tend to use:

- short options for super-common commands where everyone probably already knows what they mean
- long options if it's a little obscure AND the long option is at least a little informative (honestly, `--location` is just meaningless)

just changed this without even consciously noticing :P

---

[`Location`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Location) is the name of the header that contains the link to follow on 300 redirects. Even if my nerdy self finds it meaningful, it's definitely more obscure than other possible alternatives like `--follow-redirects` 🤓

---

> _short options for super-common commands where everyone probably already knows what they mean_

👍  It looks like a sensible default.